### DISCUSSION TITLE
How to restart EJBCA

### DISCUSSION THREAD

**Discussion by XYQ202** [Keyfactor/ejbca-ce#73] [Q&A] [answered]
Hello ，Team

Do you support rebooting? Why do I turn off Ubuntu and then turn on Ubuntu, I can't access the CA UI, is there any script that can start EJBCA?

**Comment by primetomas** [comment, marked as answer]
This has to do with starting WildFly on system start up. Yes it can be done rather easily. Here is a small piece of documentation for that.
https://doc.primekey.com/ejbca/ejbca-installation/application-servers/wildfly-24#WildFly24-StartWildFlyonSystemBoot
